cdist/conf/type/__directory/gencode-remote

```diff
@@ -57,18 +57,18 @@ get_current_value() {
 }
 
 set_group() {
-    echo chgrp $recursive \"$1\" \"$destination\"
-    echo chgrp $recursive $1 >> "$__messages_out"
+    echo "chgrp '$recursive' '$1' '$destination'"
```

Comment:
This will fail if `$recursive` is empty.

Reply:
Unquoted `$recursive` (safe, since it only ever contains nothing or `-R`, and is only set by us).